api/src/migrations/associate-records-with-people.js

```diff
@@ -27,7 +30,9 @@ const getContact = (contactId, clinicId) => {
   if (!contactId) {
     return getClinic(clinicId);
   }
-  return db.medic.get(contactId).catch(err => {
+  return dataContext
+    .bind(Contact.v1.get)(Qualifier.byUuid(contactId))
+    .catch(err => {
```

Comment:
Same goes here.  Need to update this `catch` to be a `then` with a null-check before calling `getClinic`.